packages/fxa-payments-server/src/App.tsx

```diff
@@ -55,7 +51,7 @@ export const App = ({
   return (
     <AppContext.Provider value={appContextValue}>
       <AppErrorBoundary>
-        <StripeProvider apiKey={config.stripe.apiKey}>
+        <StripeProvider apiKey={apiClient.config.stripe.apiKey}>
```

Comment:
Hmm, I don't think we should move `config` into the new `apiClient`. There are things using `config` that shouldn't ever make API calls, and I think `apiClient` should mainly be used by just the Redux store right now.